README.md

```diff
@@ -1,13 +1,17 @@
 # Project Name
 
-Replace this readme with your own information about your project.
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+Chatbot project - pair programming.
+To interact in chat using Js by adding variables, functions, eventlisternes, conditionals etc. 
```

Comment:
Try to specify what you did in the project by avoiding using words such as "etc" so the code reviewer know what you have done all in all.